docs/topics/api/reviews.rst

```diff
@@ -40,13 +40,20 @@ user has already posted a review for the current version of an add-on.
 
 .. _review-filtering-param:
 
-   By default, the review list API will only return not-deleted reviews. You
-   can change that with the ``filter=with_deleted`` query parameter, which
-   requires the Addons:Edit permission.
-
-   Only non-empty reviews (reviews with both a rating and text body) are returned,
-   unless a) the reviews were created by the user making the API request or
-   b) the user has the Addons:Edit permission.
+   By default, the review list API will only return not-deleted reviews, and
+   include reviews without text. You can change that with the ``filter`` query
+   parameter.  You can filter by multiple values, e.g. ``filter=with_deleted,without_textless``
```

Comment:
`with_textonly` is confusing though, since the default is to include reviews without text.